arches/app/views/graph.py

```diff
@@ -234,9 +234,8 @@ def get_ontology_namespaces(self):
         return ontology_namespaces
 
     def get(self, request, graphid):
-
         if graphid == settings.SYSTEM_SETTINGS_RESOURCE_MODEL_ID:
-            if not request.user.groups.filter(name="System Administrator").exists():
+            if not group_required("System Administrator", raise_exception=True):
                 raise PermissionDenied
```

Comment:
I am not sure I follow.  If the requested graphid is something other than the Systems Settings Resource Model ID, then the group need not be System Administrator - so I am not sure how this would be used in a decorator.  I see how the raise is duplicative, though.

Reply:
Oh you're quite right, I missed that when squinting at this in the files tab.